auth/id_token_from_impersonated_credentials.go

```diff
@@ -32,34 +31,35 @@ func getIdTokenFromImpersonatedCredentials(w io.Writer, scope, targetAudience, i
 	// scope := "https://www.googleapis.com/auth/cloud-platform"
 	// targetAudience := "http://www.example.com"
 	// impersonatedServiceAccount := "name@project.service.gserviceaccount.com"
-
 	ctx := context.Background()
 
-	// Construct the GoogleCredentials object which obtains the default configuration from your
+	// Construct Credentials which obtains the default configuration from your
```

Comment:
nit: put in lower case. I don't think this needs to be a proper noun.

Reply:
I can switch it to lower. The reason I has this is upper as it is the struct type: auth.Credentials.